tests/test_misc.py

```diff
@@ -39,7 +39,7 @@ class TestSubModel(BaseXmlModel, tag='model'):
 
     class TestModel(BaseXmlModel, tag='model'):
         model: TestSubModel
-        list: List[TestSubModel] = []
+        list: List[TestSubModel] = element(default=[])
```

Comment:
What is the reason of that change?

Reply:
Previously, that model would only have one possible text field, as `List` fields were implicitly elements. Now, without explicitly making a `List` type an element, it will be interpreted as the text. That would make that model have two Python fields that represent XML texts, `model` and `list`. Since the intention of the test was to have `list` represent an XML element, I changed to explicitly be so.

This is one of the backwards-incompatible changes I mentioned in the old PR.